e2e/internal/e2e/imageverify.go

```diff
@@ -163,12 +163,18 @@ func DefinitionImageVerify(t *testing.T, cmdPath, imagePath string, dfd DefFileD
 		if !fs.IsFile(file) {
 			t.Fatalf("unexpected failure: %%Pre generated file %v does not exist on host", file)
 		}
+		if err := os.Remove(file); err != nil {
+			t.Fatalf("could not remove %s: %s", file, err)
+		}
```

Comment:
Why are we removing the files here? Shouldn't this be the responsibility of whoever built the container?

Reply:
Those are canary files created on host filesystem from a generated `pre`/`setup` sections to ensure those sections was correctly executed, I don't think those files serves other purpose than checking their presence and assume the script section did its job

Reply:
This a workaround to correctly execute fakeroot profile after the root profile in `buildDefinition`, because root creates file in `filepath.Join(c.env.TestDir, "PreFile1")` and it's not removed for the second run with fakeroot profile, so the test fail with a permission denied